src/ast/variables/ExportShimVariable.ts

```diff
@@ -3,6 +3,8 @@ import { MISSING_EXPORT_SHIM_VARIABLE } from '../../utils/variableNames';
 import Variable from './Variable';
 
 export default class ExportShimVariable extends Variable {
+	module: Module;
+
 	constructor(module: Module) {
```

Comment:
One thing we could use more is to use contractor argument declarations for fields, i.e. here

```
constructor(public module: Module) {...}
```

We are already using it in some places like Chunk where it saves us quite a bit of mindless copying